exonum-java-binding/common/src/main/java/com/exonum/binding/common/proofs/CheckedProof.java

```diff
@@ -33,11 +33,11 @@
   ProofStatus getProofStatus();
 
   /**
-   * Returns the calculated root hash of the proof.
-   * Must be equal to the Merkle root hash of the collection, providing this proof.
+   * Returns the calculated index hash of the proof.
+   * Must be equal to the index hash of the collection, providing this proof.
    * @throws IllegalStateException if the proof is not valid
    */
-  HashCode getRootHash();
+  HashCode getIndexHash();
```

Comment:
I believe it would be great to notice such changes somewhere for the future "migration guide".

Reply:
Updated the changelog